### DISCUSSION TITLE
Segmentation position

### DISCUSSION THREAD

**Discussion by kknntt21** [simnibs/simnibs#349] [7.Creating head models]
Hi all,

Segmentation seems to be working, but coordinates and rotation are not set properly.
What parameters do I need to set?
![スクリーンショット 2024-05-29 113433](https://github.com/simnibs/simnibs/assets/170503814/ae50f962-35e0-43f3-8e01-5bea8db6f2d6)

**Comment by oulap** [comment]
Hmm, it looks like your input scan might be rotated somehow. If you open the input scan and the segmentation in an external viewer, for example fsleyes or freeview, do they overlap there?
